src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -166,7 +168,7 @@ internal AuthenticationResult(MsalAccessTokenCacheItem msalAccessTokenCacheItem,
         /// </example>
         public string CreateAuthorizationHeader()
```

Comment:
Shouldn't we also expose a string property with the Token Type (AuthorizationHeaderPrefix) ?
I'm thinking supportability here, if customers don't realize they got a Pop token (because they called a method, and have not realized the pop modifier was used) and try to feed it into an Authorization header as a bearer token. Under the debugger they would immediately see that this is a Pop token?

Reply:
Agreed